src/main/java/at/ac/tuwien/damap/rest/PersonResource.java

```diff
@@ -21,19 +28,58 @@
 public class PersonResource {
 
     @Inject
-    PersonService personService;
+    ConfigResource config;
+
+    LinkedHashMap<String, PersonService> personServices = new LinkedHashMap<String, PersonService>();
+
+    public PersonResource(ConfigResource config) {
+        config.personServiceConfigurations.getConfigs().forEach(serviceConfig -> {
+            try {
+                Class<?> clazz = Class.forName(serviceConfig.getClassName());
```

Comment:
Cannot find class outside its package, e.g. in a project adding this one as dependency, as our customisation project would.

Reply:
issue with my maven version